core/java/com/google/instrumentation/stats/ViewDescriptor.java

```diff
@@ -81,6 +79,43 @@ private ViewDescriptor(
   }
 
   /**
+   * Visitor for the ViewDescriptor type.
```

Comment:
yeah I think because it is an inner type of ViewDescriptor it is implicit that it is a visitor for that. Anyway, no biggie just was wondering if there was a requirement to doc all symbols.